packages/x-scheduler/src/material/styles/components/buttons.css

```diff
@@ -0,0 +1,145 @@
+.mui-x-scheduler .Button {
```

Comment:
the `mui-x-scheduler` class will probably be different before the release, at least for the material version, but at least they are in the same place now